store/test-store/tests/chain/ethereum/manifest.rs

```diff
@@ -369,6 +369,44 @@ specVersion: 0.0.2
     })
 }
 
+#[tokio::test]
+async fn parse_data_source_with_end_block() {
+    const YAML: &str = "
+dataSources:
+  - kind: ethereum/contract
+    name: Factory
+    network: mainnet
+    source:
+      abi: Factory
+      startBlock: 9562480
+      endBlock: 9562481
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.4
+      language: wasm/assemblyscript
+      entities:
+        - TestEntity
+      file:
+        /: /ipfs/Qmmapping
+      abis:
+        - name: Factory
+          file:
+            /: /ipfs/Qmabi
+schema:
+  file:
+    /: /ipfs/Qmschema
+specVersion: 0.0.2
+";
+
+    let manifest = resolve_manifest(YAML, SPEC_VERSION_0_0_4).await;
+    // Check if end block is parsed correctly
+    let data_source = manifest.data_sources.first().unwrap();
+    let end_block = data_source.as_onchain().unwrap().end_block;
+
+    assert_eq!(Some(9562481), end_block);
+    assert_eq!("Qmmanifest", manifest.id.as_str());
```

Comment:
What is this actually adding to the test?

Reply:
This was just a test i initially wrote to test the parsing of the endBlock field, if you think its not relevant i'll remove it

Reply:
No, I'm all for the test, I think that's great, I meant the assertion of Qmmanifest :)

Reply:
Yes, it's not relevant was a residue of a copy paste, removed it